linter/examples/run.yaml

```diff
@@ -0,0 +1,60 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: linter-pipeline
+spec:
+  workspaces:
+    - name: shared-workspace
+  tasks:
+    - name: fetch-repository
+      taskRef:
+        name: git-clone
+      workspaces:
+        - name: output
+          workspace: shared-workspace
+      params:
+        - name: url
+          value: https://github.com/vinamra28/blue-green-deployment-k8s
+        - name: subdirectory
+          value: ""
+        - name: deleteExisting
```

Comment:
shouldn't this follow the same naming convention use through out the task `delete-existing` ?

Reply:
@sthaha thank you...will change this :slightly_smiling_face: . Anyhow I am working on restructuring this task and breaking them into individual linters.